MANIPULATION_ENABLE: true
PICK_GROUPS: [arm] #, whole_body, whole_body_rotational
ARM_GROUP: "arm"
ARM_JOINTS: [joint1, joint2, joint3, joint4, joint5, joint6]
# ARM_GRASP: [-1.57, 0.0, 0.0, -1.57, -1.57]
ARM_GRASP: [-1.57, -0.785, -0.785, 0, 1.57, 0]
ARM_INIT: [-1.57, -1.39626, -0.174533, 0.0, 1.57, -0.7854]
ARM_HOME: [0.0, 0.0, 0.0, 0.0, 0.0, -0.7854]
ARM_PREGRASP: [-1.57, -1.04, -1.04, 0, 2.09, -2.356]
HEAD_ENABLE: false
HEAD_GROUP: "head"
HEAD_JOINTS: [Cam1Rev1, Cam1Rev2]
HEAD_LOOK_DOWN: [0.0, 0.0]
HEAD_LOOK_FRONT: [0.0, 0.0]
HEAD_LEFT: [0.0, 0.0]
HEAD_RIGHT: [0.0, 0.0]
GRIPPER_GROUP: "gripper"
GRIPPER_JOINTS: [ "Rev1Servo", "Rev2Servo"]
GRIPPER_OPEN: [-0.5, 0.5]
GRIPPER_CLOSED: [0.0, 0.0]
GRASP_POSTURES_FRAME_ID: "end_effector_link"
ALLOW_CONTACT: [
  "Servo1",
  "Servo2",
  "Dedo1",
  "Dedo2"
]
CAMERA_DEPTH_FRAME: zed2_left_camera_optical_frame
CAMERA_PC_FRAME: zed2_left_camera_frame
